src/app/search/search-filters.ts

```diff
@@ -1183,6 +1188,38 @@ function searchFilters(
       hasLight(item: DimItem) {
         return item.primStat && statHashes.has(item.primStat.statHash);
       },
+      curated(item: D2Item) {
+        if (!item || item.season <= 3) {
```

Comment:
are they only ever season <= 3, or will that change over time? Wonder if we should drive this value off of something else?

Reply:
season <= 3 did not have curated rolls so exiting early

Reply:
ahh @_@ must have not had my coffee yet. makes sense. looks good to me! 🚀